clients/src/main/java/org/apache/kafka/clients/admin/FeatureUpdate.java

```diff
@@ -24,6 +24,24 @@
 public class FeatureUpdate {
     private final short maxVersionLevel;
     private final boolean allowDowngrade;
+    private final DowngradeType downgradeType;
+
+    public enum DowngradeType {
+        UNSET(-1), // Used for backwards compatibility with allowDowngrade
```

Comment:
We shouldn't need a fourth enum value, right? The KIP just says that passing `allowDowngrade = true` to the old constructor is equivalent to setting `DowngradeType.SAFE`.